ports/sqlcipher/portfile.cmake

```
@@ -3,8 +3,8 @@ vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO sqlcipher/sqlcipher
-    REF v4.5.2
-    SHA512 1de5b219392bb976631857e32b4523258fd660fedb558d478e536b7e10c711c72c7e7c9062e45bd8a5ceaecbc1fee717935d2357f6811c3ddf76702167f4601b
+    REF v4.5.3

```

```suggestion
    REF "v${VERSION}"

```
We can use the new method. :)

---

It's not necessary, but I'll wait a day for the user to respond.